app/controllers/admin/frameworks_controller.rb

```diff
@@ -10,7 +11,6 @@ def new
   end
 
   def show
```

Comment:
FYI the `show` method is throwing a Rubocop error - "Style/EmptyMethod: Put empty method definitions on a single line"